docker/Dockerfile

```diff
@@ -76,8 +80,8 @@ RUN ln -s /usr/local/bin/install-from-source /usr/local/bin/install-header-from-
     && ln -s /usr/local/bin/install-from-source /usr/local/bin/install-from-source-with-patches
 
 # Generate toolchain files for generic
-COPY usr/local/toolchain/generate_generic_toolchain.py /usr/local/generate_toolchain.py
-RUN python /usr/local/generate_toolchain.py --prefix /usr
+RUN --mount=type=bind,source=usr/local/toolchain,target=/usr/local/toolchain \
```

Comment:
It's the same thing, just a slightly different form!
Saves a layer too

Reply:
Saving layers is important. There is a maximum layer limit of about 120 (IIRC), when you hit this limit your image will build successfully but the import of the image into docker will silently fail and it will leave you with the impression that everything finished successfully.